.bazelrc

```diff
@@ -0,0 +1 @@
+build --cxxopt='-std=c++14' --force_pic
```

Comment:
Oh I suppose if we continue to symlink `src/kj` and `src/capnp` into the Workers Runtime source tree as we do now, then of course we can override this because everything outside of `src` won't even be used in our build.

Reply:
Yes. This is only for running bazel from this tree.